templates/agent/src/app/layout.tsx

```diff
@@ -21,11 +22,13 @@ export default function RootLayout({children}: {children: React.ReactNode}) {
 		<html lang='en'>
 			<body
 				className={`${font.className} relative flex h-full min-h-screen w-full flex-col items-center`}>
-				<BackgroundGrid className='fixed h-full w-full opacity-30 dark:opacity-40' />
-				<Nav title={'> npx create-rubric-app'} />
-				<div className='z-10 flex w-full max-w-3xl items-center justify-center'>
-					{children}
-				</div>
+				<Providers>
```

Comment:
Super nice pattern

Reply:
kudos to nextjs team for releasing many open source projects to learn from